mm2src/mm2_main/Cargo.toml

```diff
@@ -20,6 +20,7 @@ run-docker-tests = []
 # TODO
 enable-solana = []
 default = []
+trezor-udp = ["crypto/trezor-udp"] # use for tests to connect to trezor emulator over udp
```

Comment:
Is this fixed? I see that it's not enabled by default now.

Reply:
Sorry, I missed this.
I tried to mark the code as `#[cfg(all(test, feature = "trezor-udp"))]` but in this case the udp support is not included when I run tests using this feature. I guess, this is because the tests are in a different crate.